### DISCUSSION TITLE
Adding border to image

### DISCUSSION THREAD

**Discussion by rizdaputra** [dolanmiu/docx#2694] [Q&A]
I am trying to add border to an image  right now it is using the css border 2px solid gray, how do i set it up for the image? tried using the border for the paragraph that wraps the image but it doesnt show. Can anyone help?

**Comment by anti-the-social** [comment]
Hi,

Have you checked this demo for images with outline?

https://github.com/dolanmiu/docx/blob/962795743c4f21d84c52e47e4f82dcf4ca536dc3/demo/5-images.ts#L67-L72
